src/site/content/en/blog/vitals-field-measurement-best-practices/index.md

```diff
@@ -0,0 +1,276 @@
+---
+title: Best practices for measuring Web Vitals in the field
+subhead: How to measure Web Vitals with your current analytics tool
```

Comment:
```suggestion
subhead: How to measure Web Vitals with your current analytics tool.
```
We've started to standardize around the convention of ending these with a period (although I didn't do it for the other Web Vitals blog post that came out recently)

Reply:
Gotcha, will keep that in mind!